### ISSUE TITLE
Need to add default value for "host" parameter as new installation crashes

### ISSUE THREAD

**Issue by ciotlosm** [danielwelch/hassio-zigbee2mqtt#483]
You need to add default value for "host" parameter, because the new installation addon is crash.

```
20-12-02 19:37:23 WARNING (MainThread) [supervisor.store.data] Can't read /data/addons/git/7ad98f9c/zigbee2mqtt/config.json: does not match regular expression for dictionary value @ data['schema']['frontend']['host']. Got 'string?'
20-12-02 19:37:23 WARNING (MainThread) [supervisor.store.data] Can't read /data/addons/git/7ad98f9c/zigbee2mqtt-edge/config.json: does not match regular expression for dictionary value @ data['schema']['frontend']['host']. Got 'string?'
20-12-02 19:37:23 WARNING (MainThread) [supervisor.store.data] Can't read /data/addons/git/7ad98f9c/zigbee2mqtt/config.json: does not match regular expression for dictionary value @ data['schema']['frontend']['host']. Got 'string?'
20-12-02 19:37:23 WARNING (MainThread) [supervisor.store.data] Can't read /data/addons/git/7ad98f9c/zigbee2mqtt-edge/config.json: does not match regular expression for dictionary value @ data['schema']['frontend']['host']. Got 'string?'
```

_Originally posted by @2vpetrov in https://github.com/danielwelch/hassio-zigbee2mqtt/issues/481#issuecomment-737454698_

**Comment by ciotlosm** [comment]
Thanks @2vpetrov. @rstein do you have time to create a fix for this?

**Comment by ciotlosm** [comment]
I will close this as a duplicate of #482 addressed in #484
